src/reminders/Reminder.ts

```diff
@@ -0,0 +1,77 @@
+import type { Moment } from 'moment';
+import { getSettings } from '../Config/Settings';
+
+export class ReminderSettings {
+    notificationTitle: string = 'Task Reminders';
+    dateFormat: string = 'YYYY-MM-DD';
+    dateTimeFormat: string = 'YYYY-MM-DD h:mm a';
```

Comment:
This puts the following text in the data.json file by default:

```json
  "reminderSettings": {
    "notificationTitle": "Task Reminders",
    "dateFormat": "YYYY-MM-DD",
    "dateTimeFormat": "YYYY-MM-DD h:mm a",
    "dailyReminderTime": "9:00 am",
    "refreshInterval": 5000
  }
```

I do not think the actual format string should be exposed in the settings. It will be an invitation for users to put arbitrary text in, and expect us to continue supporting them in future.

Reply:
I see, that is a fair point. Same suggestion as above, file for task constants. Seem brittle to keep typing 'YYYY-MM-DD'.  I see a const already exists in TaskRegularExpressions.dateFormat.

Reply:
Yes it’s true that lots of code could/should be updated to use the existing YYYY-MM-DD constant. It definitely makes sense to use the existing one.